src/main/java/Eva.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Scanner;
+
+public class Eva {
+
+    public static final String HORIZONTAL_LINE = "---------------------------------------------------------------";
+
+    public static int extractDigit(String input) {
+        String numberString = input.replaceAll("[^0-9]", "");
+        return Integer.parseInt(numberString);
+    }
+
+    public static void printTaskList(Task[] tasks, int count) {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < count; i++) {
+            System.out.println(i + 1 + ". " + "[" + tasks[i].getStatusIcon() + "] " + tasks[i].description);
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void markTaskAsDone(Task[] tasks, String line) {
```

Comment:
would markTask be a better function name? conveys the same meaning but is slightly shorter and improves readability. Same goes with unmark function below